src/job_cache/daemon_cache.cpp

```diff
@@ -596,6 +609,44 @@ int DaemonCache::run() {
   return 0;
 }
 
+void DaemonCache::remove_corrupt_job(int64_t job_id) {
+  // First remove this job from the database so that we don't get hung up on it anymore
+  impl->jobs.remove(job_id);
+
+  // Find this job directory so we can remove all the file
+  uint8_t group_id = job_id & 0xFF;
+  std::string job_dir = wcl::join_paths(wcl::to_hex(&group_id), std::to_string(job_id));
```

Comment:
I might be wrong, but I though we wrote a helper func for getting the job_dir?

Reply:
We did not thus far. I don't want to go re-factor other code to do that. I looked at doing it just now and it was a bit less than trivial since both the group_id and the job_dir wind up getting used in a couple places.